static/app/views/discover/savedQuery/utils.spec.tsx

```diff
@@ -11,7 +11,7 @@ describe('SavedQueries API helpers', () => {
   const api = new MockApiClient();
   const organization = TestStubs.Organization();
 
-  const errorsQuery = ALL_VIEWS.find(view => view.name === 'Errors by Title');
+  const errorsQuery = ALL_VIEWS.find(view => view.name === 'Errors by Title')!;
```

Comment:
not sure about using `!` 🤔 is there another approach we can use?

Reply:
I think we're OK. This is just extracting out a specific query from this constant. If it was every to change the test will fail anyway.